src/protocol/StreamrBinaryMessageV29.js

```diff
@@ -0,0 +1,95 @@
+const StreamrBinaryMessage = require('./StreamrBinaryMessage')
+const StreamrBinaryMessageV28 = require('./StreamrBinaryMessageV28')
+
+const SIGNATURE_TYPE_NONE = 0
+const SIGNATURE_TYPE_ETH = 1
+
+function hexToBytes(hex) {
+    const hexToParse = hex.startsWith('0x') ? hex.substr(2) : hex
+    let bytes
+    let c
+    for (bytes = [], c = 0; c < hexToParse.length; c += 2) {
+        bytes.push(parseInt(hexToParse.substr(c, 2), 16))
+    }
+    return bytes
+}
+
+function bytesToHex(bytes) {
+    let hex
+    let i
+    for (hex = [], i = 0; i < bytes.length; i++) {
+        hex.push((bytes[i] >>> 4).toString(16))
+        hex.push((bytes[i] & 0xF).toString(16))
+    }
+    return `0x${hex.join('')}`
+}
+
+function bytesBuf(buf, byteArray) {
```

Comment:
Ah, this is going to be slow as hell. We *really* don't want to implement this kind stuff ourselves (applies to `hexToBytes`, `bytesToHex`, and `bytesBuf`). There must be efficient built-in methods for this.

- You can get a `Buffer` out of a hex string like this (may need to check for `0x` in the start): `new Buffer("250001000192CD0000002F6D6E742F72", "hex")`
- You can write a `Buffer` to the `BufferMaker` with `bufferMaker.string(buffer)` (I think! Looks like it's being used in `StreamrBinaryMessageV28` in `toBufferMaker`